site/content/en/docs/technical-outcomes/scalability/scale-using-vpa/scale-using-vpa.demo

```diff
@@ -0,0 +1,38 @@
+# Scale pod CPU request up & down using VPA
+
+# Get metrics usage
+wget https://github.com/kubernetes-sigs/kwok/releases/download/v0.6.0/metrics-usage.yaml
+
+# Set up Cluster
+kwokctl create cluster --enable-metrics-server --config ${WORK_DIR}/kwokctl.yaml --config ${WORK_DIR}/metrics-usage.yaml --runtime binary
+
+# Create a node
+kwokctl scale node --replicas 1 --param '.allocatable.cpu="4000m"'
+
+# Set up VPA
+git clone https://github.com/kubernetes/autoscaler.git
+kubectl apply -f ${WORK_DIR}/autoscaler/vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml
+kubectl apply -f ${WORK_DIR}/autoscaler/vertical-pod-autoscaler/deploy/vpa-rbac.yaml
+
+# Execute each on new terminal
+cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/admission-controller --kubeconfig ~/.kube/config --client-ca-file ~/.kwok/clusters/kwok/pki/ca.crt --tls-cert-file  ~/.kwok/clusters/kwok/pki/admin.crt --tls-private-key  ~/.kwok/clusters/kwok/pki/admin.key --webhook-address https://127.0.0.1 --webhook-port 8080 --register-by-url --port 8080
+cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/recommender --kubeconfig ~/.kube/config
+cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/updater --kubeconfig ~/.kube/config
```

Comment:
This part feels tricky because new terminals need to be opened. We could run each command in the background and pass the live logs to a file.
However, the documentation may need to be updated with the same step. 

While labbing using this method, after deleting the cluster, I had to find and kill the process by its port number before retrying the VPA demo.
```bash
sudo lsof -i :8942
sudo kill -9 <PID>
```